modules/azure/bastion/locals.tf

```diff
@@ -0,0 +1,18 @@
+locals {
+  env_resources = { for region in var.regions : "${var.environment}-${region.location_short}-${var.name}" => {
```

Comment:
We should move away from iterating regions in the module and instead to it outside the module.

@simongottschlag might be good to write some best practices regarding Azure Terraform?

Reply:
Agreed. We shouldn't build anything new that iterates regions in the modules since we want to move away from it: https://github.com/XenitAB/terraform-modules/issues/95